velox/functions/sparksql/specialforms/SparkCastHooks.h

```diff
@@ -27,6 +27,8 @@ class SparkCastHooks : public exec::CastHooks {
   Expected<Timestamp> castStringToTimestamp(
       const StringView& view) const override;
 
+  Expected<Timestamp> castIntToTimestamp(int64_t value) const override;
```

Comment:
Perhaps change 'value' to 'seconds'. Comment suggestion:
> // When casting integral value as timestamp, the input is treated as the number of seconds since the epoch (1970-01-01 00:00:00 UTC).